pallets/xor-fee/src/lib.rs

```diff
@@ -1293,6 +1293,27 @@ pub mod pallet {
             Self::deposit_event(Event::RemintPeriodUpdated(period));
             Ok(().into())
         }
+
+        /// Scale the multiplier for weight -> fee conversion.
+        /// multiplier = multiplier * factor
+        #[pallet::call_index(7)]
+        #[pallet::weight(<T as Config>::WeightInfo::scale_multiplier())]
+        pub fn scale_multiplier(
```

Comment:
Why `scale_multiplier`, not `set_multiplier`? Just assign new value instead of calculations is more clear way.